device_policy.py

```diff
@@ -27,14 +27,34 @@ def read_policy(self, policy_bytes):
     self.device_settings = ChromeDeviceSettingsProto()
     self.device_settings.ParseFromString(self.policy_data.policy_value)
 
+  # Convert to a format capable of being input to import_policy.
+  def convert_to_policies_json(self):
+    proto_map_path = schema_dir / "device_policy_proto_map.yaml"
+    proto_map = yaml.safe_load(proto_map_path.read_bytes())
+
+    # Reverse the order of the dictionary for simple requests.
+    proto_map_reversed = {v: k for k, v in proto_map.items()}
+    
+    # Grab the settings of the policy in dictionary format.
+    policy_settings = MessageToDict(self.device_settings, preserving_proto_field_name=True)
+
+    convertedSettings = dict()
+    for name in proto_map_reversed.keys():
+        key = proto_map_reversed[name]
+        split_name = name.split('.')
+        if split_name[0] in policy_settings:
+            if split_name[1] in policy_settings[split_name[0]]:
+                convertedSettings[key] = policy_settings[split_name[0]][split_name[1]]
```

Comment:
Why does this section use 4 space indents instead of 2 spaces?

Reply:
I actually don't know and didn't notice, sorry.